llvm/test/Transforms/InstCombine/ARM/strcpy.ll

```diff
@@ -16,7 +16,7 @@ define arm_aapcscc void @test_simplify1() {
 
 
   call arm_aapcscc ptr @strcpy(ptr @a, ptr @hello)
-; CHECK: @llvm.memcpy.p0.p0.i32
+; CHECK: @llvm.memcpy.p0.p0.i64
```

Comment:
I'm a bit confused by these changes -- this test uses a `p:32:32` data layout, so wasn't the previous `i32` overload the correct one?

Reply:
The CreateMemCpy overload in question just calls getInt64().

I'm not sure it's much of a practical issue: backend legalization will truncate the integer to the appropriate size.  But maybe we should canonicalize somewhere.